clients/src/main/java/org/apache/kafka/clients/admin/CreateAclsResult.java

```diff
@@ -41,7 +41,7 @@
      * Return a map from ACL bindings to futures which can be used to check the status of the creation of each ACL
      * binding.
      */
-    public Map<AclBinding, KafkaFuture<Void>> results() {
+    public Map<AclBinding, KafkaFuture<Void>> value() {
```

Comment:
No strong opinion here, but the the name of the class is singular, so `value()` seems appropriate.